### DISCUSSION TITLE
【全文搜索】可以直接检索文章的内容，而非仅仅标题摘要？

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#2024] [Ideas]
事实上搜索原先就支持全文，但是会经常失效：

![image](https://user-images.githubusercontent.com/15920488/205439158-6e60a1df-2e83-4b08-9b7c-a3d0b3e8b906.png)

![image](https://user-images.githubusercontent.com/15920488/205439137-7604db38-4dfe-4ba6-bd11-ee817f24ed54.png)

**Comment by JIeJaitt** [comment]
貌似现在还是做不到全文搜索，有什么办法支持嘛？
